src/lib.rs

```diff
@@ -1203,13 +1244,12 @@ impl<T, S: BaseShape> Subdivided<T, S> {
     }
 
     ///
-    /// Gets the wireframe (line strip) indices which can be used to draw the edges of the
-    /// base shape’s triangles, in their subdivided form.
+    /// Appends indices for the wireframe of the subdivided form of
+    /// the base shape’s main triangles’ edges to `buffer`.
```

Comment:
Hey, this is a similar issue to a previous review I left about unicode punctuation in lieu of ascii stuff -- would you mind changing the unicode apostrophes to the ascii ones: ' ?

Reply:
Done.